homeassistant/components/emulated_hue/__init__.py

```diff
@@ -27,46 +48,14 @@
     HueUnauthorizedUser,
     HueUsernameView,
 )
-from .upnp import DescriptionXmlView, create_upnp_datagram_endpoint
-
-DOMAIN = "emulated_hue"
+from .upnp import (
+    DescriptionXmlView,
+    UPNPResponderProtocol,
+    create_upnp_datagram_endpoint,
```

Comment:
This still needs to be moved into an external lib, but I wanted to clean this up a bit more first and I'm finally getting around to doing it.